src/CMakeLists.txt

```diff
@@ -609,6 +610,11 @@ if(MIOPEN_USE_MLIR)
         )
 endif()
 
+if(MIOPEN_ENABLE_AI_HEUR)
+    list(APPEND MIOpen_Source conv/heur/metadata.cpp)
+    list(APPEND MIOPEN_Source include/miopen/conv/heur/metadata.hpp)
+    list(APPEND MIOPEN_Source include/miopen/conv/heur/heur.hpp)
```

Comment:
Please do not add headers to MIOPEN_Source

Reply:
Took them out, didn't realize that all headers in src/include were already linked.